resources/META-INF/plugin.xml

```diff
@@ -91,15 +95,16 @@
       </ul>
       ]]>
   </change-notes>
-  <version>0.9.9</version>
+  <version>1.0.0</version>
```

Comment:
```suggestion
  <version>0.10.0</version>
```

Reply:
nope. looks like only @Sohamr can accept it.

Reply:
@Sohamr hasn't allowed maintainers to push to the branch so I can't apply the suggestion